frontend/src/scenes/feature-flags/featureFlagLogic.ts

```diff
@@ -741,10 +742,22 @@ export const featureFlagLogic = kea<featureFlagLogicType>([
                 object: { name: featureFlag.key, id: featureFlag.id },
                 callback: () => {
                     featureFlag.id && actions.deleteFlag(featureFlag.id)
+                    // Load latest change so a backwards navigation shows the flag as deleted
+                    actions.loadFeatureFlag()
                     router.actions.push(urls.featureFlags())
                 },
             })
         },
+        restoreFeatureFlag: async ({ featureFlag }) => {
+            await deleteWithUndo({
```

Comment:
might be cleaner to just do a normal edit instead of `deleteWithUndo` here.. but figured it might be nice to keep the delete + restore action handlers consistent